### ISSUE TITLE
Solution - #854

### ISSUE THREAD

**Issue by CoderOggy78** [facebookresearch/segment-anything#855]
pip install segment-anything

Or clone from official repository:
Segment Anything (by Meta AI)

Then run on your GPU.

Embedding locally with CUDA:

0.2–1 second

Instead of 20–60 seconds online

##  If Using Web Version Internally 

Ensure CUDA is available
torch.cuda.is_available()

 Move model to GPU
model.to("cuda")

 Disable gradient tracking
with torch.no_grad():
    embeddings = model(image)
